src/test/java/com/moriwaki/java10thtopic/service/FishServiceTest.java

```diff
@@ -101,4 +101,13 @@ class FishServiceTest {
         verify(fishMapper, times(1)).checkById(999);
     }
 
+    @Test
+    public void データの削除処理テスト() {
+        Fish fish = new Fish(1, "ウナギ", 5322, 13);
+        doNothing().when(fishMapper).delete(1);
+        doReturn(Optional.of(fish)).when(fishMapper).checkById(1);
```

Comment:
細かいですが、こういうのは実際Serviceクラス内で呼び出される順番で書いてあげると、読み手の認知負荷が小さくなるかと思います！
```suggestion
        doReturn(Optional.of(fish)).when(fishMapper).checkById(1);
        doNothing().when(fishMapper).delete(1);
```

Reply:
もちろんこのままでも問題ないので、修正不要です！(ゆくゆくはこういった細かいところまで気を使っていただければ🙏)

Reply:
順番の修正を行いました！
https://github.com/Moriwaki-9082/Java-10th-topic/pull/11/commits/6767f2a5b102f18ed7ae4dd4621756381935e816#diff-d0c75862bb2830bcfe6dc658869b4630c4f038d988f9e6bd4a23876b723783bfL107-R108